reposerver/src/main/scala/com/advancedtelematic/tuf/reposerver/target_store/S3TargetStoreEngine.scala

```diff
@@ -82,8 +82,17 @@ class S3TargetStoreEngine(credentials: S3Credentials)(implicit val system: Actor
       TargetRedirect(Uri(signedUri.toURI.toString))
     }
   }
-}
 
+  override def delete(repoId: RepoId, filename: TargetFilename): Future[Unit] = {
+    val storagePath = storageFilename(repoId, filename)
```

Comment:
This could throw an exception.

Reply:
But everything could throw an exception? What should I do here ?

Reply:
Would be better for it to return the exception inside the Future, not outside.